apps/emqx/src/emqx_limiter/src/emqx_htb_limiter.erl

```diff
@@ -103,6 +108,8 @@
 
 -import(emqx_limiter_decimal, [sub/2, mul/2, floor_div/2, add/2]).
 
+-elvis([{elvis_style, no_if_expression, disable}]).
```

Comment:
As stone mentioned before, we try to keep if statements out of the way. So, we shouldn't add this elvis macro here.